src/bin/pgcopydb/pgsql_utils.h

```diff
@@ -0,0 +1,9 @@
+/*
+ * src/bin/pgcopydb/pgsql_utils.h
+ *     Common helper functions for interacting with a postgres server
+ */
+
+#include "libpq-fe.h"
+
+bool is_response_ok(PGresult *result);
+bool clear_results(PGSQL *pgsql);
```

Comment:
Maybe add a comment line that says where the functions are implemented? I know modern IDEs allow to easily follow that, but I like to help human code readers first.